lib/susedistribution.pm

```diff
@@ -749,7 +749,9 @@ sub activate_console {
                 return;
             }
         }
-        assert_screen "text-logged-in-$user", 60;
+        # For poo#81016, need enlarge wait time for aarch64.
+        my $waittime = check_var('ARCH', 'aarch64') ? 120 : 60;
+        assert_screen "text-logged-in-$user", $waittime;
```

Comment:
Just a minor comments:
Why not use `assert_screen("text-logged-in-$user", $waittime);` ?
Others are LGTM.

Reply:
No special reason, just keep the old code style.